tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -4107,7 +4107,7 @@ declare namespace ts {
     function idText(identifierOrPrivateName: Identifier | PrivateIdentifier): string;
     function symbolName(symbol: Symbol): string;
     function getNameOfJSDocTypedef(declaration: JSDocTypedefTag): Identifier | PrivateIdentifier | undefined;
-    function getNameOfDeclaration(declaration: Declaration | Expression): DeclarationName | undefined;
+    function getNameOfDeclaration(declaration: Declaration | Expression | undefined): DeclarationName | undefined;
```

Comment:
Not sure how I feel about changing the public API on these, but maybe it's fine.

Reply:
I think it's fine: they just make the function easier to call, and they're on functions that already return `undefined` or `boolean`, which implies (correctly) that they pass through `undefined` if you call them.